### ISSUE TITLE
Write user guide

### ISSUE THREAD

**Issue by cbeams** [bisq-network/bisq#91]
- Write user guide appropriate to features and use cases available in v0.1.
- See #169 regarding publishing the guide to docs.bitsquare.io when complete
- Work is being done in the `userguide` branch

UPDATE: see https://github.com/bitsquare/bitsquare/wiki/User-Guide

**Comment by cbeams** [comment]
@ManfredKarrer, @bitsquare/core-team: I'd like to discuss exactly what we need to have in the user guide for the 0.1 release. The original approach I had been taking (at https://docs.bitsquare.io/0.1.0-SNAPSHOT/userguide/) is probably too ambitious... it certainly didn't work to try to explain how to bring up two instances locally, etc... but we've come a long way since then.

Let's try to outline the bare essentials here:

I think that
- 1. Introduction, and
- 2. Getting Started

Are probably good to keep

We should include something about enabling UPnP and how to set up manual port forwarding. Maybe in a "Troubleshooting" section?

We should include something that explains to the user that Bitsquare is testnet-only right now, and that the only use case implemented is placing an offer to buy bitcoin and then taking an offer to buy bitcoin.

We should also explain that arbitration and account registration are not yet implemented / disabled at the moment, but that they'll be coming in future milestones.

We should include the roadmap, or a link to it.

Basically, we should communicate that Bitsquare 0.1 is a proof of concept that allows early users to test drive the process of buying bitcoin on testnet, and get a sense of where we're trying to go with the project, so they can provide feedback.

What else would you like to see (or change from the above)?

**Comment by ManfredKarrer** [comment]
@cbeams Can we postpone that a bit after the crowd funding preparation is done (Lighthouse).
I can also forward that task to Marc and Mike. Maybe they can start to fill up missing parts (like FAQ). 

I think we might also reconsider the IRC model we use atm. But lets move it to next week if its ok for you?
Re: manual port forwarding I just added in the wiki an entry
Re: Roadmap: I drafted a webpage entry (https://bitsquare.io/?page_id=304&preview=true in progress)
